include/revng/PipeboxCommon/Concepts.h

```diff
@@ -89,35 +89,54 @@ concept IsAnalysis = requires(T &A) {
 namespace detail {
 
 template<typename T>
-struct PipeRunTraits {};
+concept HasContainerTypes = SpecializationOf<typename T::ContainerTypes,
+                                             TypeList>;
```

Comment:
`Has`? `IsContainerList` maybe?

Reply:
IMHO `Has` makes sense, it's asserting that `T` has a using of `ContainerTypes` and that it's a specialization of `TypeList`

Reply:
Ah "has" in the sense of "provides".
I need to take another look.